packages/react-dom/src/__tests__/ReactDOMInput-test.js

```diff
@@ -750,6 +753,11 @@ describe('ReactDOMInput', () => {
     aNode.checked = false;
     expect(cNode.checked).toBe(true);
 
+    // The original 'checked' attribute should be unchanged
+    expect(aNode.getAttribute('checked')).toBe('');
+    expect(bNode.getAttribute('checked')).toBe(null);
+    expect(cNode.getAttribute('checked')).toBe('');
+
```

Comment:
Sorry, I think one last change. This looks good overall. 

Can you change this test to use `hasAttribute` instead of doing a value check on `getAttribute`? Like:


```javascript
expect(aNode.hasAttribute('checked')).toBe(true);
```

Reply:
That makes sense, Updated.